src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -2339,10 +2339,11 @@ class MavlinkStreamHILControls : public MavlinkStream
 			mavlink_hil_controls_t msg = {};
 
 			msg.time_usec = hrt_absolute_time();
+			// HIL FW mixer AETRFG
 			msg.roll_ailerons = out[0];
 			msg.pitch_elevator = out[1];
-			msg.yaw_rudder = out[2];
-			msg.throttle = out[3];
+			msg.yaw_rudder = out[3];
```

Comment:
That is the code i tested with. It is the wrong message that is changed. I just tested again for good measure and it does not work.